Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -209,6 +211,23 @@ extension AlertManager {
 // MARK: Extensions
 
 extension FileManager {
+    
+    fileprivate func ensureDirectoryExists(_ directory: URL?, log: DiagnosticLog) {
+        guard let directory = directory else {
+            return
+        }
+        do {
+            var isDirectory: ObjCBool = false
```

Comment:
Why not just replace this entire function with this in the AlertManager?
```
do {
  try fileManager.createDirectory(at: alertStoreDirectory, withIntermediateDirectories: true, attributes: nil)
} catch let error {
  log.error("Could not create alert store directory: %{public}@", error)
}
```

Reply:
Because this one is a little more thorough?  Not sure I understand the question.

Reply:
What is more thorough? I must be missing something.